packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,381 @@ components:
+        status:
+          type: string
+          description: Status of the asset
+          example: ready
+        originTaskId:
+          type: string
+          description: Task ID of the task that created the asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        relatedTasks:
+          type: array
+          description: List of task IDs associated with the asset
+          items:
+            $ref: "#/components/schemas/task"
+        sourceAssetId:
+          type: string
+          description:
+            ID of the source asset (root) - If missing, this is a root asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        nft:
+          type: array
+          description: List of NFTs minted with the asset as a source
+          items:
+            type: object
+            additionalProperties: false
+            properties:
+              chain:
+                type: string
+                description: Chain of the NFT
+                example: eth
+              contract:
+                type: string
+                description: Contract of the NFT
+                example: 0x0000000000000000000000000000000000000000
+              token:
+                type: number
+                description: Token id of the NFT
+                example: 123
+
+    task:
+      type: object
+      table: task
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          readOnly: true
+          description: Task ID
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        kind:
+          type: string
+          description: Type of the task
+          example: import
+        status:
+          type: string
+          description: Status of the task
+          example: ready
+        createdAt:
+          type: string
+          readOnly: true
+          description: Timestamp (in milliseconds) at which task was created
+          example: 1587667174725
+        updatedAt:
+          type: string
+          description: Timestamp (in milliseconds) at which task was updated
+          example: 1587667174725
+        params:
+          type: object
+          additionalProperties: false
+          description: Parameters of the task
+          properties:
+            import:
+              type: object
+              additionalProperties: false
+              description: Parameters for the import task
+              properties:
+                url:
+                  type: string
+                  description: URL of the asset to import
+                  example: https://cdn.livepeer.com/ABC123/filename.mp4
+            export:
+              type: object
+              additionalProperties: false
+              description: Parameters for the export task
+              properties:
+                kind:
+                  type: string
+                  description: Type of the export
+                  example: arweave|ipfs|s3
```

Comment:
Consider an `enum` as well, I guess we'll have to implement specific support for any additional protocol anyway, so could have the supported ones enumerated here as well.

Reply:
Also, let's keep the enum to only `s3` for now if that's the only one we are going to support.